src/screens/Inventory/AddInventory.tsx

```diff
@@ -171,17 +175,17 @@ function AddInventory(props: AddInventoryProps) {
         )}
         <TextField
           required
-          placeholder="e.g. 5"
+          placeholder={intl(words.eg_x, "5")}
           unit={formik.values.unit}
           type="number"
-          label={'Starting Amount'}
+          label={intl(words.starting_amount)}
           id={'startingAmount'}
           value={formik.values.startingAmount}
           onChange={formik.handleChange}
           error={formik.touched.startingAmount && Boolean(formik.errors.startingAmount)}
           helperText={formik.touched.startingAmount && formik.errors.startingAmount}
```

Comment:
I think the `helperText` value `formik.errors.startingAmount` might also need to be translated? @wangannie to make sure.

Reply:
Yeah, a brief explanation of what `formik` is doing and the role it plays would be great